modules/auth/ops/entry.sh

```diff
@@ -51,7 +51,7 @@ then
 
 else
   echo "Starting node in dev-mode"
-  exec ./node_modules/.bin/nodemon \
+  exec nodemon \
```

Comment:
Not liking this change at all. Is this necessary?

Reply:
no